install/lib/class.updater.php

```diff
@@ -127,7 +130,7 @@ public function run()
             );
 
             $this->render(new UpdaterPage('uptodate'));
-        } elseif (version_compare($currentSymphony, '2.7.0', '<')) {
+        } elseif (\Composer\Semver\Comparator::lessThan($currentSymphony, '2.7.0')) {
```

Comment:
Should 2.7.0 be replaced by 3.0.0 ? @nitriques not sure here

Reply:
No this check it there to make sure you are running at least 2.7.0 when updating to 3.0.0.

Reply:
:+1: